parser/src/test/java/com/facebook/coresql/parser/sqllogictest/java/SqlLogicTestUtils.java

```diff
@@ -0,0 +1,108 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.facebook.coresql.parser.sqllogictest.java;
+
+import com.facebook.coresql.parser.AstNode;
+import com.facebook.coresql.parser.sqllogictest.java.SqlLogicTest.SqlLogicTestError;
+
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.nio.file.FileAlreadyExistsException;
+import java.nio.file.Paths;
+import java.text.DecimalFormat;
+
+import static com.facebook.coresql.parser.ParserHelper.parseStatement;
+import static com.facebook.coresql.parser.Unparser.unparse;
+import static com.facebook.coresql.parser.sqllogictest.java.SqlLogicTest.SqlLogicTestError.PARSING_ERROR;
+import static com.facebook.coresql.parser.sqllogictest.java.SqlLogicTest.SqlLogicTestError.UNPARSED_DOES_NOT_MATCH_ORIGINAL_ERROR;
+import static com.facebook.coresql.parser.sqllogictest.java.SqlLogicTest.SqlLogicTestError.UNPARSING_ERROR;
+import static com.facebook.coresql.parser.sqllogictest.java.SqlLogicTestConfig.RESULTS_PATH;
+import static java.nio.file.Files.createFile;
+
+public class SqlLogicTestUtils
```

Comment:
Why don't we just keep these files as private static methods in the SqlLogicTest class? They are not used in any other class right?

Reply:
Yes, these aren't used anywhere else. 

Moved them into SqlLogicTest as private static methods.

Doen.